### PR TITLE
feat: Applied the new colors

### PR DESCRIPTION
**Pull request by mgaseta**
# Description

Updated the Carbon's color tokens to use the new colors from BC Gov design system.
Jira ticket: https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-176

## Type of change

- [X] Design change (change the look and feel, but no new functionality)

# How Has This Been Tested?

Tested on localhost.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/theme/tags.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  // tag-background-purple: colors.$violet-60,
+  // tag-color-purple: colors.$violet-10,
+  // tag-hover-purple: colors.$violet-50,
+  // tag-background-red: colors.$red-80,
+  // tag-color-red: colors.$red-10,
+  // tag-hover-red: colors.$red-70,
+  // tag-background-teal: colors.$teal-90,
+  // tag-color-teal: colors.$teal-10,
+  // tag-hover-teal: colors.$teal-80,
+  // tag-background-gray: colors.$gray-7,
```
Comment (mmarsoleta) [inline src/theme/tags.scss:54]: There is a typo here: 
//tag-background-gray: colors.$gray-70, 

Could you fix that? thanks! :)

**Review by mmarsoleta — CHANGES_REQUESTED** [review]
I caught just one typo in the comments of dark theme tag sections.

File: src/theme/tags.scss (hunk 1 above)
Comment (mgaseta) [reply src/theme/tags.scss]: Fixed!

**Review by RMCampos — APPROVED** [review]
@mgaseta very good job!

**Pull request merged**